corehq/apps/app_manager/models.py

```diff
@@ -5143,7 +5143,7 @@ def make_build(self, comment=None, user_id=None, previous_version=None):
         if not copy._id:
             # I expect this always to be the case
             # but check explicitly so as not to change the _id if it exists
-            copy._id = copy.get_db().server.next_uuid()
+            copy._id = uuid.uuid4().hex
```

Comment:
I'm guessing you thought of this, but did you check that these are the same format ("look the same") as the ones coming from couch. Would be mildly annoying to have a visible change in the ids like them previously having dashes and now not having dashes or something.